src/components/list-item/list-item.tsx

```
@@ -0,0 +1,400 @@
+                <div class="text">
+                    {this.renderLabel()}
+                    {this.renderDescription()}
+                </div>
+                {this.renderActionMenu(this.actions)}
+            </Host>
+        );
+    }
+
+    private renderLabel = () => {
+        return <label id={this.labelId}>{this.text}</label>;
+    };
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)